test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1242,6 +1254,59 @@ def expected_bigquery_complex_manifest(self):
                     'patch_path': self.dir('bq_models/schema.yml'),
                     'docrefs': [],
                 },
+                'model.test.multi_clustered': {
+                    'alias': 'multi_clustered',
+                    'config': {
+                        'cluster_by': ['first_name','email'],
+                        'column_types': {},
+                        'enabled': True,
+                        'materialized': 'table',
+                        'partition_by': 'updated_at',
+                        'post-hook': [],
+                        'pre-hook': [],
+                        'quoting': {},
+                        'vars': {}
+                    },
+                    'depends_on': {'macros': [], 'nodes': ['seed.test.seed']},
+                    'empty': False,
+                    'fqn': ['test', 'multi_clustered'],
+                    'name': 'multi_clustered',
+                    'original_file_path': multi_clustered_sql_path,
```

Comment:
This isn't defined anywhere -- can you define `multi_clustered_sql_path` up above next to `clustered_sql_path`?